### ISSUE TITLE
index 0 is out of bounds error for a questionable signomial constraint

### ISSUE THREAD

**Issue by pgkirsch** [hoburg/gpkit#610]
```python
from gpkit import Variable, Model, SignomialsEnabled

x = Variable('x')
y = Variable('y', 1)
z = Variable('z', 4)
with SignomialsEnabled():
    m = Model(x, [x + z >= y])

m.localsolve()
```
I'm not sure how you would like to handle this case (arguably the constraint is valid, even if pretty dumb), but I think it would be good to at the very least have a useful error message, instead of the current:

```python
IndexError: index 0 is out of bounds for axis 0 with size 0
```

**Comment by bqpd** [comment]
:+1: issue title

**Comment by whoburg** [comment]
fixed by #611
